zb-adapter.js

```diff
@@ -793,6 +793,12 @@ class ZigbeeAdapter extends Adapter {
     for (let i = 0; i < frame.numEntriesThisResponse; i++) {
       const neighborIndex = frame.startIndex + i;
       const neighbor = frame.neighbors[i];
+      if (neighbor.addr64 == '0000000000000000') {
+        const node = this.findNodeByAddr16(neighbor.addr16);
```

Comment:
You're shadowing the outer `node` variable here. Could you use a different name for clarity?